src/Altinn.Platform/Altinn.Platform.Register/Altinn.Platform.Register.Tests/TestingControllers/OrganizationsControllerTests.cs

```diff
@@ -47,8 +49,11 @@ public async Task GetOrganization_ValidTokenRequestForExistingOrganization_Retur
             HttpClient client = GetTestClient(organizationsService.Object);
             client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
 
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, "/register/api/v1/organizations/" + orgNo);
+            httpRequestMessage.Headers.Add("PlatformAccessToken", token);
```

Comment:
I think we should consider having separate tokens for the two different authentication/authorization mechanisms. It would make it more obvious that they are separate systems and ensure that we can detect any weird bug where they influence each other.

Reply:
Yes. Changed to seperate tokens